### ISSUE TITLE
New web interface "Transactions" error

### ISSUE THREAD

**Issue by davidepatti** [raspiblitz/raspiblitz#3270]
![Screenshot from 2022-08-01 23-25-03](https://user-images.githubusercontent.com/3337669/182249534-267da20a-f7ad-45f0-b9d2-42c421e58955.png)

Everythings else seems just fine.

**Comment by redmac68** [comment]
+1
![image](https://user-images.githubusercontent.com/48153702/182251326-b11b2cf9-51c6-4584-8184-3e2463810a20.png)

**Comment by rootzoll** [comment]
👀 @fusion44

**Comment by fusion44** [comment]
Looks like a gRPC error which might originate from the gRPC server.

@davidepatti @redmac68 Do you run CLN or LND? How many transactions do you roughly have?

**Comment by redmac68** [comment]
I run LND.
In July 2255 transactions.

**Comment by davidepatti** [comment]
> Looks like a gRPC error which might originate from the gRPC server. 
> @davidepatti @redmac68 Do you run CLN or LND? How many transactions do you roughly have?

I use LND, since december 2021, usually like 200/300 transactions per month

**Comment by cstenglein** [comment]
@davidepatti @redmac68  Does this still occur?

**Comment by Sebe0401** [comment]
Having the same problem... is there a solution already?

**Comment by rootzoll** [comment]
@Sebe0401 do you still have this issue with v1.11.0?

**Comment by Sebe0401** [comment]
> @Sebe0401 do you still have this issue with v1.11.0?

@rootzoll yes, it only shows a rotating circle and then this message pops up. 
![20240426_223653](https://github.com/raspiblitz/raspiblitz/assets/143658735/4ad2e20b-3e2f-49da-9c73-954c523b01fa)

Maybe i have too many transactions? I am doing many rebalance requests. These requests normally showed up there.

**Comment by rootzoll** [comment]
@fusion44 can you take a look again?

**Comment by fusion44** [comment]
I will, but I'm not certain that there is a good solution, except upstream letting me query for only a portion of the data at a time. Maybe things have changed since I've last looked.
